src/ReTestItems.jl

```diff
@@ -865,17 +878,18 @@ function runtestitem(
             push!(body.args, :(const $setup = $ts_mod))
         end
         @debugv 1 "Setup for test item $(repr(name)) done$(_on_worker())."
+
+        # add the `test_end_expr` to a module to be run after the test item
+        test_end_body = copy(body)
```

Comment:
Hmmm....I guess that gets a little tricky, because the `test_end_expr` can't really express that it needs test setups apart from the test items. It doesn't feel like we should auto-import setups into the test_end_expr though; like, the `test_end_expr` should be stand-alone and any imports you need, you need to put them there explicitly.

Reply:
ah, good catch. i just did `copy` to save duplicating code, but all i wanted was to get `using Test` and `using PackageBeingTested` included -- i can move this to not also import the test-setups

Reply:
changed to not import test-setups